cli/command.go

```diff
@@ -110,8 +123,42 @@ func RunCMD() cli.Command {
 				}
 
 				if wait {
+					logDone := make(chan struct{})
+					go func() {
+						defer recovery.LogStackTraceAndContinue("log handling thread")
+						defer close(logDone)
+						timer := time.NewTimer(0)
+						defer timer.Stop()
+
+						for {
+							select {
+							case <-ctx.Done():
+								grip.Notice("operation canceled")
+								return
+							case <-timer.C:
+								logLines, err := client.GetLogStream(ctx, cmdID, 1000)
+								if err != nil {
```

Comment:
Should this be resilient to a transient failure?

Reply:
I think the clients can/do have middleware for doing retries for network errors and the like, so it's not useful doing the retries here.